code/datums/traits/negative.dm

```diff
@@ -619,12 +619,12 @@
 	if(world.time > next_process)
 		next_process = world.time + process_interval
 		var/deleted = QDELETED(reagent_instance)
-		if(deleted || !LAZYFIND(H.reagents.addiction_list, reagent_instance))
+		if(deleted || !(LAZYFIND(H.reagents.addiction_list, reagent_instance)))
```

Comment:
you should really wrap the define properly rather then doing it only here

Reply:
Not sure if I got this 100% right, but fixed?